scenarios/01_Highest_Tiers.cfg

```diff
@@ -292,7 +293,7 @@
             x=18
             y=40
             facing=se
-            placement,overwrite=map,yes
+            placement,overwrite,invulnerable=map,yes,yes
```

Comment:
dont like that you put unrelated attribute to same line

Reply:
fair